src/wms/Database.ts

```diff
@@ -60,7 +70,35 @@ export class Database {
     if (query.length === 0) {
       throw new EvalError('No daily schedules found in WMS, check coneection or content');
     }
-    return query as StaffSchedule[];
+
+    return query.map<StaffSchedule>((q) => {
+      this.validate(q.C_ID, 'C_ID');
+
+      logger.info(`Processing C_ID: ${q.C_ID}`);
+
+      this.validate(q.STAFF_ID, 'STAFF_ID');
+      this.validate(q.STATUS, 'STATUS');
+      this.validate(q.EVENT_DATE, 'EVENT_DATE');
+      this.validate(q.EVENT_START, 'EVENT_START');
+      this.validate(q.EVENT_END, 'EVENT_END');
+
+      const schedule: StaffSchedule = {
+        c_id: q.C_ID,
+        staff_id: q.STAFF_ID,
+        status: q.STATUS,
+        event_date: q.EVENT_DATE.split(' ')[0], // We only want the date part from "YYYY-MM-DD 00:00:00"
```

Comment:
Are we sure this is always the format? Just asking because ISO is `T` separated: `yyyy-MM-ddTHH:mm:ss`

Reply:
yes but it's just making sure that the `event_date: q.EVENT_DATE.split(' ')[0],` can potentially handle a `YYYY-MM-DDTHH:MM:SS` date object in the case the `YYYY-MM-DD HH:MM:SS` format from the sql db gets reformatted.

Reply:
It seems to come through like this: 2022-02-24 00:00:00. I was surprised too!